test/unit/Breadcrumbs.test.js

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-env mocha */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+import Breadcrumbs, { getBreadcrumbLinks, getCrumbTrail } from '../../src/app/components/Breadcrumbs/Breadcrumbs';
+
+import { initialSearchQuery } from '../../src/app/stores/InitialState';
+
+describe('Breadcrumb', () => {
+  const searchQuery = { queries: [{ query: 'snakes', field: 'keyword' }] };
+  const workDetail = { uuid: '12345' };
+  const homeLocation = { pathname: '/' };
+  const otherLocation = { pathName: '/other-location' };
+
+  describe('getBreadcrumbLinks', () => {
```

Comment:
would love your thoughts on whether this is too overzealous.  I think some white-box testing is necessary because it doesn't make sense for us to bundle design system and SFR and test them all as a whole, but it might be less brittle to test the contents of the Breadcrumbs render in the L104 block rather than testing these methods separately.  Like, if you were a dev trying to modify breadcrumbs, would these tests be helpful or annoying?

Reply:
I think having both of these tests are helpful. The separate method unit tests give clarity that the methods are performing exactly as anticipated. For me that means that they can help catch edge cases etc. I think that the white-box functional testing is good to verify that the component renders as expected but for me the unit tests are just as important for preventing regressions, the "brittle-ness" might be a good thing since we should be able to very precisely predict those outputs.